ui/app/pages/swaps/swaps.util.js

```diff
@@ -265,6 +285,28 @@ export async function fetchTokenBalance (address, userAddress) {
   return usersToken
 }
 
+export async function fetchSwapsGasPrices () {
+  const gasPricesUrl = getBaseApi('gasPrices')
+  const response = await fetchWithCache(gasPricesUrl, { method: 'GET' }, { cacheRefreshTime: 15000 })
+  const responseIsValid = validateData(SWAP_GAS_PRICE_VALIDATOR, response, gasPricesUrl)
+
+  if (!responseIsValid) {
+    return {}
```

Comment:
This case should throw an error I think? We should treat this scenario similarly to a HTTP error response; throw and let the caller deal it somehow.

Reply:
Done in bbf5b5090